docs/content/en/docs/community/contributing.md

```diff
@@ -5,80 +5,129 @@ description: >
-to your bug report or contribution effectively.
+Please read through this document before submitting any issues or pull requests to ensure we have all the necessary
+information to effectively respond to your bug report or contribution.
+
+## General Guidelines
+
+### Pull Requests
+
+Make sure to keep Pull Requests small and functional to make them easier to review, understand, and look up in commit history.
+This repository uses "Squash and Commit" to keep our history clean and make it easier to revert changes based on PR.
+
+Adding the appropriate documentation, unit tests and e2e tests as part of a feature is the responsibility of the
+feature owner, whether it is done in the same Pull Request or not.
+
+Pull Requests should follow the "subject: message" format, where the subject describes what part of the code is being
+modified.
+
+Refer to the [template](https://github.com/aws/eks-anywhere/blob/main/.github/PULL_REQUEST_TEMPLATE.md) for more information on what goes into a PR description.
+
+### Design Docs
+
+A contributor proposes a design with a PR on the repository to allow for revisions and discussions.
+If a design needs to be discussed before formulating a document for it, make use of GitHub Discussions to
+involve the community on the discussion.
+
+### GitHub Discussions
+
+GitHub Discussions are used for feature requests (that don't have actionable items/issues), questions, and anything else
+the community would like to share.
+
+Categories:
+* Q/A - Questions
+* Proposals - Feature requests and other suggestions
+* Show and tell - Anything that the community would like to share
+* General - Everything else (possibly announcements as well)
+
+### GitHub Issues
+
+GitHub Issues are used to file bugs, work items, and feature requests with actionable items/issues (Please refer to the
+"Reporting Bugs/Feature Requests" section below for more information).
+
+Labels:
+* "\<area\>" - area of project that issue is related to (create, upgrade, flux, test, etc.)
+* "priority/p\<n\>" - priority of task based on following numbers
+  * p0: need to do right away
+  * p1: don't have a set time but need to do
+  * p2: not currently being tracked (backlog)
+* "status/\<status\>" - status of the issue (notstarted, implementation, etc.)
+* "kind/\<kind\>" - type of issue (bug, feature, enhancement, etc.)
+
+Refer to the [template](https://github.com/aws/eks-anywhere/tree/main/.github/ISSUE_TEMPLATE) for more information on
+what goes into an issue description.
+
+### GitHub Milestones
+
+GitHub Milestones are used to plan work that is currently being tracked.
+
+* next: changes for next release
+* next+1: won't make next release but the following
+* techdebt: used to keep track of techdebt items, separate ongoing effort from release action items
+* oncall: used to keep track of issues needing active follow-up
+* backlog: items that don't have a home in the others
+
+### GitHub Projects (or tasks within a GitHub Issue)
```

Comment:
@jaxesn are we keeping the roadmap in this repo? If so we should link to it from the main containers-roadmap repo and update the blog link.

Reply:
Plan is to have milestones immediately, but not opposed to a roadmap